templates/logstash.service.j2

```diff
@@ -0,0 +1,28 @@
+# {{ ansible_managed }}
```

Comment:
It's really bad practice to hardcode variables. Add them to defaults and put here

Reply:
This group should allow access to logstash to use system monitor files, I believe this is necessary for some internal reasons.